clients/vault/src/metrics.rs

```diff
@@ -1,27 +1,1341 @@
-use std::collections::HashMap;
+use std::{collections::HashMap, convert::TryInto, sync::Arc};
 
+use crate::{
+	// execution::parachain_blocks_to_bitcoin_blocks_rounded_up,
+	system::{VaultData, VaultIdManager},
+	Error,
+};
+use async_trait::async_trait;
+// use bitcoin::{json::ListTransactionResult, GetTransactionResultDetailCategory, SignedAmount,
+// TransactionExt};
+use futures::{try_join, StreamExt, TryFutureExt};
```

Comment:
apparently these imports are unused, can you remove them? And also the commented imports

Reply:
Done